docker/jammin-as-lan.Dockerfile

```
@@ -0,0 +1,47 @@
+    rm -rf /var/lib/apt/lists/*
+
+# Pin to 0.8.0 with --locked for reproducible builds.
+# $CARGO_HOME is /usr/local/cargo in the official rust image, so the
+# resulting binary lands at /usr/local/cargo/bin/wasm-pvm.
+RUN cargo install wasm-pvm-cli@0.8.0 --locked
+
+# ---- Stage 2: runtime ----------------------------------------------------
+FROM node:22-slim AS runtime
+
+# ca-certificates: npm registry TLS. git: npm deps pointing at git URLs.
+RUN set -eux; \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)